src/components/Person/Person.jsx

```diff
@@ -1 +1,20 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const partner = person.sex === 'm'
+    ? 'wife'
+    : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+      {person.age && (<p className="Person__age">{`I am ${person.age}`}</p>)}
+      {person.isMarried && (
+        <p className="Person__partner">{`${person.partnerName} is my ${partner}`}</p>
+      )}
+      {!person.isMarried && (
+        <p className="Person__partner">I am not married</p>
+      )}
+    </section>
```

Comment:
```suggestion
export const Person = ({ person }) => {
  const partner = person.sex === 'm'
    ? 'wife'
    : 'husband';

  return (
    <section className="Person">
      <h2 className="Person__name">
        {`My name is ${person.name}`}
      </h2>
      
      {person.age && (
        <p className="Person__age">
          {`I am ${person.age}`}
        </p>)}
        
      {person.isMarried && (
        <p className="Person__partner">
          {`${person.partnerName} is my ${partner}`}
        </p>
      )}
      
      {!person.isMarried && (
        <p className="Person__partner">
          I am not married
        </p>
      )}
    </section>
  );
};
```
To make your code more readable, It's better to write code like in the code above.
Also, you used twice "isMarried" here, try to combine these two blocks into one using ternary operators.